### ISSUE TITLE
Doesn't work properly on OpenBSD and I have no idea why

### ISSUE THREAD

**Issue by DrHyde** [DrHyde/perl-modules-App-ptimeout#5]

**Comment by eserte** [comment]
Not quite. The same test failures happen also on other systems, but not in 100% of all cases:
http://matrix.cpantesters.org/?dist=App-ptimeout+1.0.2

**Comment by DrHyde** [comment]
All of the non-OpenBSD cases I've looked into were caused by the testing machine being under heavy load and so some timeouts in the tests being exceeded. But with OpenBSD it _always_ fails even on unloaded machines.

**Comment by DrHyde** [comment]
Fixed in c7dfb73
